crates/ruff_python_formatter/src/string/implicit.rs

```diff
@@ -154,7 +154,7 @@ impl<'a> FormatImplicitConcatenatedStringFlat<'a> {
             }
 
             // Multiline strings can never fit on a single line.
-            if !string.is_fstring() && string.is_multiline(context.source()) {
+            if !string.is_fstring() && string.is_multiline(context) {
```

Comment:
Do we still need the check on line 190-204 considering that it's now covered by `string.is_multiline`?

Reply:
I think not, I can remove it.